core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -826,6 +826,12 @@ public boolean isExternalTable() {
     return external != null && external.equalsIgnoreCase("true");
   }
 
+  public boolean isFileLevelExternalTable() {
```

Comment:
why is this property required?

Reply:
**stored by 'carbondatafileformat' is mapped with _filelevelexternal.**
So, In carbonScanRDD, when mapReduce service or hadoop service calls carbonScanRDD.
based on _filelevelexternal, new File level reader [CarbonFileInputFormat] will be called.

External table can be table level (stored by 'carbondata') or file level (stored by 'carbondatafileformat')
**This is used to identify the file level external table.**